imports/utils.lua

```diff
@@ -0,0 +1,1007 @@
+local isServer = IsDuplicityVersion()
```

Comment:
Consider putting a comment at the top of this file that summarizes how it's used, and what's in it.

Reply:
I don't think that's a good idea as the modules that I have ready will change the system up a bit, so you would import a different file that handles the importing of these files.